bin/sweepj2

```diff
@@ -29,30 +29,24 @@ args = vars(parser.parse_args())
 
 if not os.path.exists(args['output']):
     os.makedirs(args['output'])
-
+pwd = os.getcwd()
 ext = os.path.splitext(args['template'])[1]
+if not os.path.isfile(args['template']):
+    print("ERROR: File {} does not exist".format(os.path.join(pwd, args['template'])))
```

Comment:
this assumes that the path given in `--template` is always relative, which might not be true always. For example, one could do `--template /tmp/script.sh`. It is OK to use in the message the exact value that users are passing